packages/gtk_doc.rb

```diff
@@ -29,12 +29,11 @@ class Gtk_doc < Package
   depends_on 'docbook'
   depends_on 'docbook_xml'
   depends_on 'docbook_xsl'
+  depends_on 'six'
```

Comment:
Can we remove the pre-built binaries and bump the version?  Does the six package fix so this compiles cleanly now?

Reply:
Actually, there is an update available.  Upgrade to https://github.com/GNOME/gtk-doc/archive/GTK_DOC_1_28.tar.gz?

Reply:
No. Version bump won't matter, crew recursively checks dependencies anyway.
This package is cr&p.

Reply:
If this package is not needed to get code working, I would rather it be removed from this PR and submitted separately.

Reply:
Dude, you completely changed the build section.  This needs new binaries.

Reply:
Maybe. I'll try it.

Reply:
Try upgrading to version 1.28.  Always gotta think about moving forward. :)

Reply:
Nope.

Reply:
Fixed. No more binaries for docbook_xsl, docbook_xml, or docbook ever.

Reply:
I still see binaries.

Reply:
It works now. Try it.

Reply:
Dude, there is no difference between binaries and compiling from source.  Well, the difference is binary installs are a LOT faster but we're not gonna compile these each time.  That doesn't make sense.